site/en/docs/react_gemini_prompting.ipynb

```diff
@@ -0,0 +1,973 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.      """finishes the conversation on encoutring  token by
</pre><p>Typo: encoutring</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>